wurst/systems/crafting/QuickMakeSpell.wurst

```diff
@@ -353,10 +351,10 @@ function getQuickMakeSpells() returns LinkedList<QuickMakeSpell>
         new QuickMakeSpell("Omnitower Kit"         , TRANSMUTE_CRAFT_BUILDING_TOOLTIP+OMNI_TOWER_KIT_RECIPE        , "OrcTower"             , SPELL_QM_OMNI_TOWER_KIT        , 1, 2, "X", QM_BUILD_TOOLIP, "warstomp"),
 
         // Second Buildings Page
-        new QuickMakeSpell("Smoke House Kit"       , TRANSMUTE_CRAFT_BUILDING_TOOLTIP+SMOKE_HOUSE_KIT_RECIPE       , "RockTower"            , SPELL_QM_SMOKE_HOUSE_KIT       , 0, 0, "Q", QM_BUILD_TOOLIP, "divineshield"),
-        new QuickMakeSpell("Storage Hut Kit"       , TRANSMUTE_CRAFT_BUILDING_TOOLTIP+STORAGE_HUT_KIT_RECIPE       , "ForestTrollHut1"      , SPELL_QM_STORAGE_HUT_KIT       , 1, 0, "W", QM_BUILD_TOOLIP, "summongrizzly"),
-        new QuickMakeSpell("Teleport Beacon Kit"   , TRANSMUTE_CRAFT_BUILDING_TOOLTIP+TELEPORT_BEACON_KIT_RECIPE   , "EnergyTower"          , SPELL_QM_TELEPORT_BEACON_KIT   , 2, 0, "E", QM_BUILD_TOOLIP, "defend"),
-        new QuickMakeSpell("Hatchery Kit"          , TRANSMUTE_CRAFT_BUILDING_TOOLTIP+HATCHERY_KIT_RECIPE          , "SnowOwl"              , SPELL_QM_HATCHERY_KIT          , 3, 0, "R", QM_BUILD_TOOLIP, "summonwareagle")
+        // new QuickMakeSpell("Smoke House Kit"       , TRANSMUTE_CRAFT_BUILDING_TOOLTIP+SMOKE_HOUSE_KIT_RECIPE       , "RockTower"            , SPELL_QM_SMOKE_HOUSE_KIT       , 0, 0, "Q", QM_BUILD_TOOLIP, "divineshield"),
```

Comment:
Just delete these. No point in keeping them. Only comment out code you expect to use long term but isn't currently working.